### DISCUSSION TITLE
Make a button to set pid parameters

### DISCUSSION THREAD

**Discussion by ronaldom12000** [artisan-roaster-scope/artisan#663] [Q&A]
Hi! I am using artisan software and the integrated  PID to control my roaster by arduino and SSR. Everything is fine, but I need do make a button to set the pid parameters on the control tab in Artisan. I tried set Artisan Command and action, but I made something wrong and it didn't work. My request is because I found two PID settings (p, i, d), one works fine from begin to 190°, and another works fine from first crack to the end. Then I want to autommated this change in pid settings during the roast, by alarme action trigger by temperature, and a button, but I don't know how to set this button.

**Comment by hermetheuscoffee** [comment]
This is a late answer, but you just have to set up two buttons as shown below (but use whatever values you want for the p, i, and d values instead of the ones I use:
![PID_values](https://user-images.githubusercontent.com/95500043/146451032-89babc87-9fc5-4f21-84e8-883edfed563d.png)

Hope this helps!
-Jason
